Classes/DDOSLogger.m

```diff
@@ -17,6 +17,12 @@
 
 #import <os/log.h>
 
+@interface DDOSLogger ()
+@property (copy, nonatomic, readwrite) NSString *subsystem;
```

Comment:
These don't have to be `readwrite` anymore. They're readonly.

Reply:
@ffried Do you mean ivars or logger initialization in designed initializer?

Reply:
@lolgear ivars. I just checked, DDFileLogger also uses ivars in the initializer.